### ISSUE TITLE
Permission problems should be more descriptive

### ISSUE THREAD

**Issue by meshuggahtas** [TASVideos/tasvideos#913]
https://tasvideos.org/Privileges should be linked in the error text at least as a starting point

1. Find a Privilege you don't have
2. Try to access it
3. Here's a Site Developer privilege required page https://tasvideos.org/Diagnostics which says: 
Access denied
You do not have access to this resource.

Outdated text, when Editor didn't had Edit Ram Addresses role:
```
1. https://tasvideos.org/Addresses-List Add button
2. https://tasvideos.org/Addresses-List click on an entry, View button

Don't know what's the requirement, but Editor was not enough
```

**Comment by adelikat** [comment]
I don't think I want this to be too descriptive.  I feel like stating exactly what permissions you are missing opens up more potential for exploitation.

You should not be able to get to one of those by pointing and clicking on the site, if so, those situations need to be documented.  Outside of that, you are a "misbehaving user" and I'm not too concerned if you are getting less helpful error messages.

**Comment by meshuggahtas** [comment]
Personally I'm okay with this decision. 

However... the point of descriptive error messages in general is to understand why the problem happens. While in this case I only suggested it as the current Privileges list is a long uncategorized page that doesn't seems easy to grasp, I had to create an issue ticket to know what was my original problem to get an answer. If this question I had cannot be made convenient enough to the end user, then I don't know how would any potential hacker benefit from a more descriptive error message for an open source project.
